frontcamp-patterns/components/newsapi/newsapi.proxy.js

```diff
@@ -0,0 +1,11 @@
+"use strict";
+
+import NewsApi from "./newsapi.module";
+
```

Comment:
Good idea will be to implement in Proxy class mechanism of caching  result s of requests.